tests/integration/targets/test_zabbix_action/tasks/main.yml

```diff
@@ -334,7 +334,7 @@
     register: zbxaction_conditions_reorder
 
   - assert:
-      that: zbxaction_conditions_reorder.changed is sameas True
+      that: zbxaction_conditions_reorder.changed is sameas False
```

Comment:
Lol I wonder how this was not failing tests...

Reply:
Please see my previous comments. On fresh containers and first run it will not fail but later it will start failing. And we are talking about using this collection in real life so if order does not matter than the resource should not be indicated as 'changed' -)